src/ContactPage/Contact.scss

```diff
@@ -2,45 +2,51 @@
 @import "../styles/layout";
 
 .contact-section {
+  background-image: url(../../assets/letters.jpg);
+  
+  
+  height:calc(100vh - 60px);
```

Comment:
So i can't see the background but i'm guessing it has the same issue as the one on ur portfolio page. So i'd recommend again making this height 100% if it doesn't look good on mobile